server/app/services/MessageKey.java

```diff
@@ -105,7 +105,7 @@ public enum MessageKey {
   BUTTON_PREVIOUS_SCREEN("button.previousScreen"),
   BUTTON_PRINT("button.print"), // North Star only
   BUTTON_BACK("button.back"),
-  BUTTON_SAVE_AND_EXIT("button.saveAndExit"),
+  BUTTON_REVIEW_AND_EXIT_V2("button.reviewAndExit.v2"),
```

Comment:
There isn't another reviewAndExit, right? So no need for the v2 here?

Reply:
Correct. I'll remove the v2. 👍